package.json

```diff
@@ -57,7 +57,6 @@
   "dependencies": {
     "babel-polyfill": "^6.26.0",
     "brace": "^0.10.0",
-    "c3": "^0.4.11",
```

Comment:
Yayyy!! :+1: 

patternfly-react are using react-c3js:
https://github.com/patternfly/patternfly-react/blob/master/package.json#L25

react-c3js using c3:
https://github.com/bcbcarl/react-c3js/blob/master/package.json#L39

Considered we just introduced the `webpack.vendor.js`:
https://github.com/theforeman/foreman/blob/develop/config/webpack.vendor.js

I think we should be able to remove the c3 from the `webpack.vendor.js` and let pattenrfly-reat to be the only consumer of this package.
Can you try to remove it and check if it's still work? also need to check if the `c3` is getting into the `vendor.js` naively.
You can verify it with:
https://gist.github.com/sharvit/19440366d4edcb3c4302b19e8049b7be